### DISCUSSION TITLE
Nextflow Error - Unexpected input: '{'

### DISCUSSION THREAD

**Discussion by selenaajohn** [nextflow-io/nextflow#3659] [Q&A] [answered]
I'm trying to run my nextflow pipeline, and im keep getting this error. I've tried making changes to my code but cant seem to figure out what the issue is. Ive looked at other discussions and assume I have a sytnax error. Any suggestions on what my error might be would be greatly appreciated!

<img width="587" alt="Screen Shot 2023-02-17 at 12 21 15 PM" src="https://user-images.githubusercontent.com/97220964/219767854-a42040ae-36f6-4202-af92-20efb0e604f5.png">

<img width="697" alt="Screen Shot 2023-02-17 at 12 27 19 PM" src="https://user-images.githubusercontent.com/97220964/219768348-7f2c30d4-6467-4c08-9f23-fb339f94513c.png">
<img width="692" alt="Screen Shot 2023-02-17 at 12 22 04 PM" src="https://user-images.githubusercontent.com/97220964/219768416-9bb2a411-5585-48d1-a30e-fbce9b52ef95.png">

**Comment by mribeirodantas** [comment, marked as answer]
This error indeed suggests a syntax error. The issue in your process seems to be in the output block: You should write `path("*.sam")` instead of `path(*.sam)`.
